netspeed/data/org.mate.panel.applet.netspeed.gschema.xml.in

```diff
@@ -16,9 +16,9 @@
       <description>If true, show speed in bits instead of bytes.</description>
     </key>
     <key name="show-all-addresses" type="b">
-      <default>false</default>
+      <default>true</default>
       <summary>Show IPv4 and IPv6 addresses on tooltip</summary>
-      <description>If true, show both IP addresses if enabled.</description>
+      <description>If true, show both IPv4 and (if available) IPv6 addresses in tooltip.</description>
```

Comment:
I think the new description makes no sense. 

NetworkManager screenshot:

<img width="645" alt="Captura de Pantalla 2020-12-22 a les 8 27 16" src="https://user-images.githubusercontent.com/10171411/102861211-94785d80-442f-11eb-81b5-bbba551a9332.png">

Reply:
Why not? If checkbox is disabled, it will only show the IPv4 address. If the checkbox is enabled, it will also show IPv6 addresses. Thus, both the IPv4 _and_ the IPv6.

Reply:
Using `if available` to refer to IPv6 addressing seems like a joke.

Reply:
Should I change it to `if enabled`? Or just leave out the bracket?

Reply:
Yes, using `if enabled` is more meaningful. I don't care about w/- or w/o ()

Reply:
You have to keep in mind that if ipv4 isn't enabled, it won't be shown in the text either, that is the reason for placing if enabled is at the end of the sentence.